### ISSUE TITLE
Add functionality to notify user on each schedule run

### ISSUE THREAD

**Issue by deepakpathania** [deepakpathania/Noah#15]
In the create schedule endpoint, accept a slack webhook, and invoke that after each run with the response from the endpoint, in order to notify user.

**Comment by ankitjaggi** [comment]
Can this be made generic instead of targetting just Slack? There can be other ways to send notifications.

**Comment by deepakpathania** [comment]
Yes, I'd love a generic implementation for this, I thought slack was the easiest to get started with and thus marked this as a `good first issue`.

Just thinking out loud, an optional `notify` block during create schedule seems like a good way to start this. Something along the lines of 

```js
{
  "display": {
    "channel": "slack | email | push",
    "target": "webhook | email id | fcm token"
  }
}
```

Thoughts? Happy to assign this to you in case you'd like to pick it up.

**Comment by ankitjaggi** [comment]
You should also think about multiple channels at a time. There could be a use case where multiple channels can be specified as well.

**Comment by deepakpathania** [comment]
We can make `display` an array for future-proofing, with the structure specified above.
